### ISSUE TITLE
Favicon addition

### ISSUE THREAD

**Issue by raphael-khan** [raphael-khan/Home_Team#5]
Add a favicon to the title.

**Comment by AbhishekD15** [comment]
Hi @raphael-khan, can I work on this isssue.

**Comment by raphael-khan** [comment]
Yes sir, go ahead and fork the project. Let me know if you have any questions. @AbhishekD15

**Comment by AbhishekD15** [comment]
Thanks @raphael-khan. Can you assign this issue to me and grant me write request to create a PR

**Comment by AbhishekD15** [comment]
> Yes sir, go ahead and fork the project. Let me know if you have any questions. @AbhishekD15


>

**Comment by AbhishekD15** [comment]
@raphael-khan should we close this one !
